src/adapters/postgres/user-adapter.ts

```
@@ -1706,4 +1717,222 @@ export class PostgresUserService implements IServicelocator {
+        identifier = username;
+      } else {
+        return APIResponse.error(
+          response,
+          apiId,
+          API_RESPONSES.BAD_REQUEST,
+          "INVALID_REASON",
+          HttpStatus.BAD_REQUEST
+        );
+      }
+      const [hashValue, expires] = hash.split('.');
+      if (!hashValue || !expires || isNaN(parseInt(expires))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)